go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/tidwall/sjson v1.0.4
 	go.uber.org/atomic v1.4.0
 	go.uber.org/zap v1.10.0
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
```

Comment:
You have changed the `go.mod` file, but I could not see any vendors changes. Have you executed `make adaptor dummy lint` on your laptop?

Reply:
updated